### ISSUE TITLE
Support of constructor with parameters

### ISSUE THREAD

**Issue by andreibancioiu** [second-state/SOLL#4]
From the code it seems that [constuctors with parameters](https://github.com/second-state/SOLL/blob/master/lib/CodeGen/CodeGenModule.cpp#L969) are not yet supported.

Though, **no error is issued by the compiler** (`EmitLLVM`), but `llvm-link` throws an error instead.

Example (adding a parameter in the constructor of `0-0-3.sol`):

```
constructor(uint256 something) public {
	...
}
```

Error:

```
llvm-link: /home/.../0-0-3.ll:748:13: error: '@solidity.constructor.uint256' defined with type 'void (i256)*' but expected 'void ()*'
```

**Comment by hydai** [comment]
Hi, @andreibancioiu. Thanks for the report.

This is a bug. The expected behavior should be `SOLL throws an error`.

**Comment by hydai** [comment]
Hi @andreibancioiu 
Thanks for your report. We've fixed this issue at [0.0.5 release](https://github.com/second-state/SOLL/releases/tag/0.0.5).

Now, SOLL will throw an error to notify this.

<img width="1451" alt="螢幕截圖 2020-01-18 20 39 11" src="https://user-images.githubusercontent.com/2776756/72664113-092f6480-3a35-11ea-8bf7-326b107f1a29.png">
